cpp/src/Ice/ConnectionI.cpp

```diff
@@ -2167,7 +2166,7 @@ Ice::ConnectionI::initiateShutdown()
         //
         // Before we shut down, we send a close connection message.
         //
-        OutputStream os(_instance.get(), Ice::currentProtocolEncoding);
+        OutputStream os{Ice::currentProtocolEncoding};
```

Comment:
We don't create about format or string converter for CloseConnection or ValidateConnection.